coordinator-service/src/app.ts

```diff
@@ -89,6 +127,48 @@ export function initExpress({
         },
     )
 
+    app.get('/chunks/:id', (req, res) => {
```

Comment:
I think we should optimize this further and give the contributor/verifier only the data they need. I believe that we only use this endpoint to get the download URLs for the last contribution if I'm a contributor and of the last and one before last if I'm a verifier.